homeassistant/components/livisi/coordinator.py

```diff
@@ -69,14 +70,15 @@ async def async_setup(self) -> None:
                 livisi_connection_data=livisi_connection_data
             )
         controller_data = await self.aiolivisi.async_get_controller()
-        if controller_data["controllerType"] == "Avatar":
+        if controller_data["controllerType"] == AVATAR:
```

Comment:
```suggestion
        if (self.controller_type := controller_data["controllerType"]) == AVATAR:
```

Reply:
I have an error here: `cannot use assignment expressions with attribute`. Can I replace it with if (controller_type := controller_data["controllerType"]) == AVATAR: , and then assign the controller_type to self.controller_type ?

Reply:
Sound good

Reply:
I changed this, thank you.